akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Behaviors.scala

```diff
@@ -57,6 +57,21 @@ object Behaviors {
    */
   def stopped[T]: Behavior[T] = Behavior.stopped
 
+  /**
+   * Return this behavior from message processing to signal that this actor
+   * shall terminate voluntarily. If this actor has created child actors then
+   * these will be stopped as part of the shutdown procedure.
+   *
+   * The when the PostStop signal that results from stopping this actor is received
+   * the given `postStop` callback will be invoked. All other messages and signals will effectively be
+   * ignored.
+   */
+  def stopped[T](postStop: () ⇒ Unit): Behavior[T] = Behavior.stopped(receiveSignal {
+    case (_, PostStop) ⇒
+      postStop()
+      Behaviors.same
+  })
```

Comment:
ok, I probably went overboard with the principle of avoiding callbacks. This is better. Let's remove/internalize the other one that takes Behavior.